browser/main/lib/ConfigManager.js

```diff
@@ -26,7 +26,17 @@ export const DEFAULT_CONFIG = {
     toggleMain: OSX ? 'Command + Alt + L' : 'Super + Alt + E',
     toggleMode: OSX ? 'Command + Alt + M' : 'Ctrl + M',
     deleteNote: OSX ? 'Command + Shift + Backspace' : 'Ctrl + Shift + Backspace',
-    pasteSmartly: OSX ? 'Command + Shift + V' : 'Ctrl + Shift + V'
+    pasteSmartly: OSX ? 'Command + Shift + V' : 'Ctrl + Shift + V',
+    reload: OSX ? 'Command + R' : 'Ctrl + R',
+    devTool: OSX ? 'Command + Alt + I' : 'Control + Shift + I',
```

Comment:
Should we allow users to customize `devTool`?

Reply:
why not?

Reply:
Well maybe a user will change it (without knowing what it does) because it conflict with the hotkey that they want and next time they can’t open the dev tool using that hotkey anymore. But I think we can help the user in that case so I guess we can leave it :smile:

Reply:
Personally I think shortcuts only need to be available for frequently used features. Maybe an advanced settings part could be added to optionally set shortcuts to advanced features like the `devTool`.

Reply:
I'm thinking of a scenario where a user wants to use the shortcut that was assigned to `devTool` so the customizable `devTool` shortcut in this case is useful